library/alloc/tests/vec.rs

```diff
@@ -293,6 +293,7 @@ fn test_retain() {
 }
 
 #[test]
+#[cfg_attr(not(panic = "unwind"), should_panic)]
```

Comment:
Hm, this is an interesting way to annotate these. Are traps actually caught on wasm like this?

Reply:
This just tells the test harness to ignore the test on non-unwind targets, because `should_panic` tests are ignores when `panic != "unwind"`. `#[cfg_attr(not(panic = "unwind"), ignore)]` would work just as well